2-numpy/manual.ipynb

```diff
@@ -286,7 +286,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In the above code the expression `diffs%2 == 1` creates a mask of Boolean values. Only the subset of values corresponding to `True` in the mask are doubled.\n",
+    "In the above code the expression `diffs % 2 == 1` creates an array of Boolean values, a mask, which is used to index `mod_diffs`. Only the subset of values corresponding to `True` in the mask are doubled.\n",
```

Comment:
The new phrasing is better, but perhaps this should be edited even more. What we call masking, the NumPy documentation calls [Boolean array indexing](https://numpy.org/doc/stable/user/basics.indexing.html#boolean-array-indexing) instead. Our use of the word "masking"  might cause confusion with [NumPy masked arrays](https://numpy.org/doc/stable/reference/maskedarray.html). It might be a good idea to ensure the vocabulary we use is the same NumPy itself uses.

Reply:
Good point. I have renamed the sections and added a small bit of text explaining the common nomenclature.